R/plotAbundance.R

```diff
@@ -237,7 +237,7 @@ setMethod("plotAbundance", signature = c("SummarizedExperiment"), function(
 # agglomeration and transformation. The outut is a single tibble with all the
 # whole dataset for plotting. 
 .get_abundance_data <- function(
-        x, assay.type, layout, group = rank, rank = NULL,
+        x, assay.type, layout = "bar", group = rank, rank = NULL,
```

Comment:
No need for this since the layout is already declared in line 174

Reply:
@Daenarys8

Reply:
test fails  `.get_abundance_data()`  or I could just pass it directly in the test.

Reply:
You can pass directly from the test. It might be easier to see from the interval function which parameters are not yet declared i.e. they are "hidden parameters"

Reply:
okay. it's working now